mobile/app/Components/UserSignUpForm.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { View, Button, TextInput } from 'react-native';
+// import axios from 'axios';
+// USECLIENT TO CHANGE STATE !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+interface whatever {
+  onChangeUser: (email, demographics, name, phoneNum, community) => void
+}
```

Comment:
You don't need props, so you may remove this interface entirely.

Reply:
resolved